backend/src/main/java/saci/configuration/WebMvcConfiguration.java

```diff
@@ -9,15 +9,24 @@
 @Configuration
 public class WebMvcConfiguration {
 
-    @Value("${CORS_URL:http://localhost:5173}")
-    private String corsUrl;
+    @Value("${CORS_URL_CLIENT:http://localhost:8081}")
+    private String corsUrlClient;
+
+    @Value("${CORS_URL_ADMIN:http://localhost:8082}")
+    private String corsUrlAdmin;
 
     @Bean
     public WebMvcConfigurer corsConfigurer() {
         return new WebMvcConfigurer() {
             @Override
             public void addCorsMappings(CorsRegistry registry) {
-                registry.addMapping("/**").allowedMethods("*").allowedOrigins(corsUrl);
+                // Client CORS configuration: Allow GET and POST requests
+                registry.addMapping("/**")
+                        .allowedMethods("GET", "POST")
```

Comment:
Does the client need POST type requests?

Reply:
Victor requested me to add a post